charts/prometheus-postgres-exporter/templates/deployment.yaml

```diff
@@ -101,6 +101,7 @@ spec:
               port: http
           readinessProbe:
             initialDelaySeconds: {{ .Values.readinessProbe.initialDelaySeconds }}
+            timeoutSeconds: {{ .Values.readinessProbe.timeoutSeconds }}
```

Comment:
The k8s default of one second should be in the values.yaml

Reply:
Sorry im not to sure what you mean by this. Can one by default change the k8s readiness timeoutSeconds ?

Reply:
You're using a variable .Values.readinessProbe.timeoutSeconds which is not defined in values.yaml.
Therefore variable would be empty by default.
The Kubernetes standard value for timeoutSeconds is 1 second.
Therefore it should be in the values.yaml like this.